<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello Layer Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_layer_name">
     <property name="text">
      <string>Layer: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_feature_count">
     <property name="text">
      <string>Features: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="refresh_button">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
